rs/cli/Cargo.toml

```diff
@@ -77,6 +77,7 @@ comfy-table = { workspace = true }
 human_bytes = { workspace = true }
 mockall.workspace = true
 clio = { workspace = true }
+serde_yaml.workspace = true
```

Comment:
We should be able to easily switch to https://github.com/acatton/serde-yaml-ng in one of the future PRs.